examples/proto/examplepb/a_bit_of_everything.swagger.json

```diff
@@ -1556,7 +1556,186 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/examplepbABitOfEverything"
+              "type": "object",
+              "example": {
+                "uuid": "0cf361e1-4b44-483d-a159-54dabdf7e814"
```

Comment:
This is a bit unfortunate - the example here is from the proto message definition, and that example has uuid in it. Arguably, the example should be updated since it's not really applicable to everywhere the message is used.

Reply:
That sounds right - could you update the example to include both the required values `int64_value` and `double_value`?